scripts/eurostat/health_determinants/tobacco_consumption/process.py

```diff
@@ -0,0 +1,847 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script Load the datasets, cleans it
```

Comment:
This sentence needs revision:

The Python script loads the datasets, cleans them and generates the cleaned CSV, MCF and TMCF files.

If this is just downloading one dataset, then everything in the sentence above should be singular.

Reply:
Modified the comment.